commit d6bc8bc6de1cd21f86076063cdd905b57dd1007c
Author: Peter Hartmann <phartmann@rim.com>
Date:   Thu Nov 22 17:29:04 2012 +0100

    BlackBerry mkspecs: tell compiler the specific ARM version (cortex-a9)
    
    ... so the compiler knows which instructions to generate and can
    optimize for the specific CPU.
    All BlackBerry platforms are Cortex-A9 compatible.
    
    (cherry picked from commit 44b1c5dde2dfbb69a29cbd4ad8d1f0ac0203b482)
    
    Change-Id: I730dea78421d4688e0afa60f8f095f3166e5b733
    Reviewed-by: Sean Harmer <sean.harmer@kdab.com>

diff --git a/mkspecs/blackberry-armv7le-qcc/qmake.conf b/mkspecs/blackberry-armv7le-qcc/qmake.conf
index 5edcf0e..a48d8fe 100644
--- a/mkspecs/blackberry-armv7le-qcc/qmake.conf
+++ b/mkspecs/blackberry-armv7le-qcc/qmake.conf
@@ -16,3 +16,5 @@ contains(QT_CONFIG, stack-protector-strong) {
 } else {
     QMAKE_CFLAGS += -fstack-protector -fstack-protector-all
 }
+
+QMAKE_CFLAGS += -mcpu=cortex-a9
